libraries/core-react/src/components/Accordion/AccordionItem.tsx

```diff
@@ -1,4 +1,5 @@
-import * as React from 'react'
+import { Children as ReactChildren, cloneElement } from 'react'
```

Comment:
Looks like you ended up with double import statements from `react` (line 1 and line 9) here, so suggest changing it to just one :)